docs/pages/guides/authentication.mdx

```diff
@@ -2054,6 +2055,63 @@ export default function App() {
 </Box>
 {/* End Uber */}
 
+{/* Start Logto */}
+
+<Box
+  name="Logto"
+  createUrl="https://docs.logto.io/quick-starts/expo/"
+  image={ASSETS.logto}
+>
+
+| Website                    | Provider | PKCE      | Auto Discovery |
+| -------------------------- | -------- | --------- | -------------- |
+| [Get Your Config][c-logto] | OpenID   | Supported | Available      |
```

Comment:
```suggestion
| [Get Your Config](https://docs.logto.io/quick-starts/expo/) | OpenID   | Supported | Available      |
```

Let's add the link here since it is only used once in this section.